apps/aecore/test/aecore_forking_SUITE.erl

```diff
@@ -278,7 +278,20 @@ setup_node(N, Top, Epoch, Config) ->
     delete_file(filename:join(RelD, "sys.config.orig")),
     TestD = filename:join(filename:dirname(code:which(?MODULE)), "data"),
     cp_file(filename:join(TestD, "sync_SUITE.config"),
-            filename:join(DDir , "sync_SUITE.config")).
+            filename:join(DDir , "sync_SUITE.config")),
+    cp_genesis_dir(DDir, Epoch).
+
+cp_genesis_dir(DDir, Epoch) ->
```

Comment:
Shall this be rather added in aec_test_utils?

Reply:
Yes, it would be better, I'm on it